TodoList/TodoList/Controllers/UsersController.cs

```diff
@@ -40,21 +41,20 @@ public async Task<ActionResult<Respond>> SignUp(Register user)
                 });
             }
         }
-        /*[HttpPost("login")]
-        public async Task<ActionResult> Login(Register user)
+        [HttpPost("login")]
+        public async Task<IActionResult> Login(Login user)
```

Comment:
```suggestion
        public async Task<ActionResult<Respond>> Login(Login user)
```
để như thế này thì khi generate ra swagger ms có body của response